data/hai/hai reveal 6 end.txt

```diff
@@ -49,9 +49,23 @@ mission "Hai Reveal [C01] Cleanup"
+			`Arriving on Mountaintop, you see the station has seen serious renovations, including a grand ballroom, all manner of diplomatic suites, and a notable lack of movie props. Rather than actors and stage crew, you see many familiar and unfamiliar people in full diplomatic regalia. The Chancellor of the Free Worlds and President of the Republic are in their most respectable suits, Alondo and Xilin have the traditional symbols of office (though Alondo's looks like the Free Worlds have perhaps not finalized their design just yet), and dress uniforms laden with medals abound on both sides. You appear to be a guest of honor, being occasionally called upon to answer only a few idle queries from people seeking clarity on one point or another - and otherwise allowed to mingle with anyone taking a break from their duties. Throughout these preliminary meetings, you notice Remington and Terry are conspicuously absent among the attendees.`
+			choice
+				`	(Ask Xilin about it.)`
+				`	(Ignore it.)`
+					goto publicbroadcast
+			branch presaside
+				has patronizedthepresident
+			`	Between scheduled sessions, Xilin explains the Syndicate's absence. "The Syndicate is technically part of the Republic and awarded no special diplomatic status. Besides, Terry seems to have disappeared from Syndicate communications, though I won't hazard a guess as to their internal politics."`
+				goto publicbroadcast
+			label presaside
+			`	At one quiet point between structured meetings, during which the peripheral conversations are more focused on the particulars of the Free World's position, you manage to get a hold of Xilin to ask about it. He explains the Syndicate's absence. "The Syndicate is technically part of the Republic and awarded no special diplomatic status. Besides, Terry seems to have disappeared from Syndicate communications, though I won't hazard a guess as to their internal politics."`
+			`	Perhaps seizing on the opportunity to get a momentary reprieve from more official business, the president himself steps in to fill the gap as soon as Xilin departs. It seems that you acquire a bubble of space around you whenever anyone of note stands next to you and the president exploits this. "You know," he says quietly, while staring out at the many people hustling about at their business, "it's not very often that someone has the gall to patronize me to my face... I usually have to pay people to do that, or at least, to do it constructively anyway."`
```

Comment:
```suggestion
			`	Perhaps seizing on the opportunity to get a momentary reprieve from more official business, the President himself steps in to fill the gap as soon as Xilin departs. It seems that you acquire a bubble of space around you whenever anyone of note stands next to you, and the President exploits this. "You know," he says quietly, while staring out at the many people hustling about at their business, "it's not very often that someone has the gall to patronize me to my face... I usually have to pay people to do that, or at least, to do it constructively anyway."`
```
I think that's title, not role, here, so needs capitalized. Also, am I the only one reading `I usually have to pay people to do that` coming across less as an official job offer and more as a... proposition?

Reply:
The ambiguity is intentional. ;)